libvast/src/system/spawn_importer.cpp

```diff
@@ -52,7 +55,7 @@ maybe_actor spawn_importer(node_actor* self, spawn_arguments& args) {
   }
   for (auto& source : self->state.registry.find_by_type("source")) {
     VAST_DEBUG("{} connects source to new importer", self);
-    self->send(source, atom::sink_v, caf::actor_cast<caf::actor>(handle));
+    self->anon_send(source, atom::sink_v, caf::actor_cast<caf::actor>(handle));
```

Comment:
Why anonymously?

Reply:
Can't send non-anon messages from a typed actor to an untyped actor.

Reply:
Is this a bug or a feature?

Reply:
Neither. It's just how it needed to be implemented, because typed actors check whether they have handlers for the reply, but they can't do so for untyped actors. The `anon_send` makes it explicit that it's not checked properly.